packages/form-js-editor/src/features/properties-panel/entries/StaticValuesSourceEntry.js

```diff
@@ -75,16 +75,33 @@ export default function StaticValuesSourceEntry(props) {
 
 // helper
 
-function getIndexedEntry(index) {
+function getIndexedEntry(index, values) {
   const entry = {
     label: 'Value',
     value: 'value'
   };
 
+  while (labelOrValueIsAlreadyAssignedForIndex(index, values)) {
+    index++;
+  }
+
   if (index > 1) {
     entry.label += ` ${index}`;
     entry.value += `${index}`;
   }
 
   return entry;
+}
+
+function labelOrValueIsAlreadyAssignedForIndex(index, values) {
+
+  if (index === 0) {
```

Comment:
I wonder whether this can ever happen, as we set `const index = values.length + 1;` above.

Reply:
You're right, this is ironically, a redundant check :trollface: